core/arch/arm/kernel/boot.c

```diff
@@ -68,6 +68,10 @@ struct ns_entry_context ns_entry_contexts[CFG_TEE_CORE_NB_CORE];
 static uint32_t spin_table[CFG_TEE_CORE_NB_CORE];
 #endif
 
+#if defined(CFG_CORE_SEL2_SPMC) && defined(CFG_CORE_PHYS_RELOCATABLE)
+paddr_t manifest_dtb __nex_data;
```

Comment:
`vaddr_t` ?

Reply:
When manifest_dtb is used, the MMU is not enabled. In this case, the address is the physical address.